contracts/protocol/tokenization/base/ScaledBalanceTokenBase.sol

```diff
@@ -76,9 +76,7 @@ abstract contract ScaledBalanceTokenBase is MintableIncentivizedERC20, IScaledBa
     require(amountScaled != 0, Errors.INVALID_MINT_AMOUNT);
 
     uint256 scaledBalance = super.balanceOf(onBehalfOf);
-    uint256 balanceIncrease = scaledBalance.rayMul(index) -
-      scaledBalance.rayMul(_userState[onBehalfOf].additionalData);
-
+    uint256 balanceIncrease = scaledBalance.rayMul(index - _userState[onBehalfOf].additionalData);
```

Comment:
This approach is not exactly equivalent to the other because of math rounding and precision. We should stick to the original approach